hazelcast-sql/src/main/codegen/templates/Parser.jj

```diff
@@ -0,0 +1,8405 @@
+/*
```

Comment:
We might add a note to the beginning where this file is copied from and how is it modified.